### DISCUSSION TITLE
脚本能否和自己的网站共存

### DISCUSSION THREAD

**Discussion by drwangyd** [mack-a/v2ray-agent#873] [General]
大神，八合一脚本安装的伪装网站，能不能换成自己需要的实际网页服务？具体怎么实施？

**Comment by mack-a** [comment]
可以的，/etc/nginx/conf.d/alone.conf 修改此文件中的31xxx端口的配置，如果是静态网页则改root的路径或者alias路径，如果是端口服务修改一下反向代理，nginx具体配置可以问一下AI。
